### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by runar-rkmedia** [t9md/atom-cursor-history#31]
Encountering errors for almost all new click in the new updated. This did not occur in the previous release of atom-cursor-history. 

1. Click anywhere in a file

**Atom**: 1.17.0-beta3 x64
**Electron**: 1.3.14
**OS**: Mac OS X 10.12.4
**Thrown From**: [cursor-history](https://github.com/t9md/atom-cursor-history) package 0.10.0

### Stack Trace

Uncaught TypeError: Cannot read property 'length' of undefined

```
At /Users/runarkristoffersen/.atom/packages/cursor-history/lib/history.coffee:131

TypeError: Cannot read property 'length' of undefined
    at History.module.exports.History.uniqueByBuffer (/packages/cursor-history/lib/history.coffee:131:27)
    at /packages/cursor-history/lib/history.coffee:11:8
    at Config.module.exports.Config.observeKeyPath (/Applications/Atom Beta.app/Contents/Resources/app/src/config.js:625:1)
    at Config.module.exports.Config.observe (/Applications/Atom Beta.app/Contents/Resources/app/src/config.js:150:1)
    at /packages/cursor-history/lib/history.coffee:10:35)
    at Object.getHistory (/packages/cursor-history/lib/main.coffee:53:21)
    at Object.checkLocationChange (/packages/cursor-history/lib/main.coffee:129:10)
    at /packages/cursor-history/lib/main.coffee:68:12
```

### Commands

```
     -4:22.5.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling)
```

### Non-Core Packages

```
aligner-python 1.0.1 
atom-alignment 0.12.1 
atom-beautify 0.29.23 
atom-clock 0.1.7 
atom-css-clean 2.19.6 
atom-dedupe-open-files 0.1.0 
atom-django 0.3.2 
atom-like-brackets-editor 1.1.4 
atom-reveal-file-in-finder 1.1.4 
atom-runner 2.7.1 
autocomplete-json 5.4.0 
autocomplete-python 1.8.63 
autoprefixer 3.6.1 
browser-plus 0.0.87 
browser-refresh 0.9.0 
browser-refresh-on-save 0.0.6 
busy-signal 1.4.1 
change-case 0.6.5 
color-picker 2.2.5 
cursor-history 0.10.0 
dash 1.7.1 
django-test-runner 0.5.0 
emmet 2.4.3 
enhanced-tabs 1.7.0 
expand-region 0.2.7 
expand-selection-to-quotes 0.11.0 
file-icons 2.1.4 
filecolor 0.4.0 
fold-comments 0.6.0 
git-plus 7.7.0 
gitignore-snippets 0.2.5 
golden-ratio 0.3.0 
goto 1.8.3 
highlight-line 0.12.0 
highlight-selected 0.13.1 
inc-dec 1.2.0 
intentions 1.1.2 
intentions-texdoc 0.3.0 
javascript-snippets 1.2.1 
keybinding-cheatsheet 0.1.1 
language-django 0.1.1 
language-latex 1.0.0 
language-lua 0.9.11 
language-markdown 0.21.0 
language-pgsql 0.2.3 
language-restructuredtext 1.1.0 
language-sql-mysql 0.3.1 
latex-hyperclick 0.2.0 
latextools 0.8.5 
linter 2.1.4 
linter-chktex 1.3.0 
linter-csslint 1.3.4 
linter-htmlhint 1.3.3 
linter-jshint 3.0.3 
linter-jsonlint 1.3.0 
linter-pylama 0.8.16 
linter-rst 0.0.3 
linter-spell 0.13.1 
linter-spell-latex 0.9.2 
linter-ui-default 1.2.3 
lipsum 0.1.0 
lorem 1.2.1 
markdown-preview-plus 2.4.9 
markdown-writer 2.6.4 
open-in-browsers 0.0.19 
open-recent 5.0.0 
pdf-view 0.57.0 
pigments 0.39.1 
preview-inline 1.4.7 
python-indent 1.0.2 
remove-whitespace 0.2.1 
rulerz 0.4.3 
script 3.14.1 
simply-django-atom 0.0.2 
sort-lines 0.14.0 
split-diff 1.3.1 
Sublime-Style-Column-Selection 1.7.4 
swackets 0.26.0 
sync-settings 0.8.1 
tab-to-escape 0.1.0 
tabularize 0.2.5 
tail 0.1.0 
text-pastry 0.6.1 
todo-show 1.11.0 
tree-view-git-modified 0.7.2 
tree-view-git-status 1.3.1 
turbo-javascript 1.4.0 
wakatime 7.0.0 
zentabs 0.8.8 
```

**Comment by t9md** [comment]
OK, thanks.
Find the cause.

You enabled `keepSingleENtryPerBuffer` right?
Will fix in next day.
Sorry for degradation.

**Comment by t9md** [comment]
fix released as v0.10.1.
